From 608d3d9ad9e2e589c47ffbc6cc28951156238765 Mon Sep 17 00:00:00 2001
From: Thomas Haller <thaller@redhat.com>
Date: Wed, 14 Aug 2013 12:14:50 +0200
Subject: [PATCH 1/1] man: backport manpages for nm-applet and nm-connection-editor (rh #564465) (rh #564467)

Manual pages were added to the upstream repository network-manager-applet in
commit 1b13f7fe38ffe447e7100d6dc8f7011352ddd772.

After that initial adding, there were some minor fixes on these files.
So this commit does not take the manual pages from 1b13f7f but instead
from the current master dbe37f874d4f058a5aeeadd838cd0c22cb3c70e0 .
---
 network-manager-applet-0.8.1/Makefile.am           |    2 +-
 network-manager-applet-0.8.1/configure.ac          |    3 +
 network-manager-applet-0.8.1/man/Makefile.am       |    2 +
 network-manager-applet-0.8.1/man/nm-applet.1.in    |   43 ++++++++++++++
 .../man/nm-connection-editor.1.in                  |   60 ++++++++++++++++++++
 5 files changed, 109 insertions(+), 1 deletions(-)
 create mode 100644 network-manager-applet-0.8.1/man/Makefile.am
 create mode 100644 network-manager-applet-0.8.1/man/nm-applet.1.in
 create mode 100644 network-manager-applet-0.8.1/man/nm-connection-editor.1.in

diff --git a/network-manager-applet-0.8.1/Makefile.am b/network-manager-applet-0.8.1/Makefile.am
index 6f164b9..d5da902 100644
--- a/network-manager-applet-0.8.1/Makefile.am
+++ b/network-manager-applet-0.8.1/Makefile.am
@@ -1,6 +1,6 @@
 ACLOCAL_AMFLAGS = -I m4 ${ACLOCAL_FLAGS}
 
-SUBDIRS = src icons po
+SUBDIRS = src icons po man
 
 EXTRA_DIST =				\
 	CONTRIBUTING			\
diff --git a/network-manager-applet-0.8.1/configure.ac b/network-manager-applet-0.8.1/configure.ac
index 51fed5c..7f1021d 100644
--- a/network-manager-applet-0.8.1/configure.ac
+++ b/network-manager-applet-0.8.1/configure.ac
@@ -130,5 +130,8 @@ icons/32/Makefile
 icons/48/Makefile
 icons/scalable/Makefile
 po/Makefile.in
+man/Makefile
+man/nm-applet.1
+man/nm-connection-editor.1
 ])
 AC_OUTPUT
diff --git a/network-manager-applet-0.8.1/man/Makefile.am b/network-manager-applet-0.8.1/man/Makefile.am
new file mode 100644
index 0000000..06ab1bb
--- /dev/null
+++ b/network-manager-applet-0.8.1/man/Makefile.am
@@ -0,0 +1,2 @@
+man_MANS = nm-applet.1 nm-connection-editor.1
+
diff --git a/network-manager-applet-0.8.1/man/nm-applet.1.in b/network-manager-applet-0.8.1/man/nm-applet.1.in
new file mode 100644
index 0000000..6e583b4
--- /dev/null
+++ b/network-manager-applet-0.8.1/man/nm-applet.1.in
@@ -0,0 +1,43 @@
+.\" nm-applet (1) manual page
+.\"
+.\" This is free documentation; you can redistribute it and/or
+.\" modify it under the terms of the GNU General Public License as
+.\" published by the Free Software Foundation; either version 2 of
+.\" the License, or (at your option) any later version.
+.\"
+.\" The GNU General Public License's references to "object code"
+.\" and "executables" are to be interpreted as the output of any
+.\" document formatting or typesetting system, including
+.\" intermediate and printed output.
+.\"
+.\" This manual is distributed in the hope that it will be useful,
+.\" but WITHOUT ANY WARRANTY; without even the implied warranty of
+.\" MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+.\" GNU General Public License for more details.
+.\"
+.\" You should have received a copy of the GNU General Public Licence along
+.\" with this manual; if not, write to the Free Software Foundation, Inc.,
+.\" 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+.\"
+.\" Copyright (C) 2013 Red Hat, Inc.
+.\"
+.TH NM-APPLET "1" "15 February 2013"
+
+.SH NAME
+nm\-applet \- network monitor and control GUI applet
+.SH SYNOPSIS
+.B nm\-applet
+.RI " [ " OPTIONS " ]
+
+.SH DESCRIPTION
+.B nm\-applet
+is a GTK\(hybased GUI applet to monitor network status and devices and to start
+and stop network connections managed by NetworkManager.  \fInm\-applet\fP
+is normally started at login by the desktop session manager and does not
+need to be run manually.  \fInm\-applet\fP conforms to the XDG System Tray
+specification and requires that the desktop environment provide a System Tray
+implementation in which the applet will be embedded.
+
+.SH SEE ALSO
+.BR NetworkManager(8),
+.BR nm\-connection\-editor(1).
diff --git a/network-manager-applet-0.8.1/man/nm-connection-editor.1.in b/network-manager-applet-0.8.1/man/nm-connection-editor.1.in
new file mode 100644
index 0000000..9a971f5
--- /dev/null
+++ b/network-manager-applet-0.8.1/man/nm-connection-editor.1.in
@@ -0,0 +1,60 @@
+.\" nm-connection-editor (1) manual page
+.\"
+.\" This is free documentation; you can redistribute it and/or
+.\" modify it under the terms of the GNU General Public License as
+.\" published by the Free Software Foundation; either version 2 of
+.\" the License, or (at your option) any later version.
+.\"
+.\" The GNU General Public License's references to "object code"
+.\" and "executables" are to be interpreted as the output of any
+.\" document formatting or typesetting system, including
+.\" intermediate and printed output.
+.\"
+.\" This manual is distributed in the hope that it will be useful,
+.\" but WITHOUT ANY WARRANTY; without even the implied warranty of
+.\" MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+.\" GNU General Public License for more details.
+.\"
+.\" You should have received a copy of the GNU General Public Licence along
+.\" with this manual; if not, write to the Free Software Foundation, Inc.,
+.\" 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+.\"
+.\" Copyright (C) 2013 Red Hat, Inc.
+.\"
+.TH NM-CONNECTION-EDITOR "1" "15 February 2013"
+
+.SH NAME
+nm\-connection\-editor \- network connection editor for NetworkManager
+.SH SYNOPSIS
+.B nm\-connection\-editor
+.RI " [ " OPTIONS " ]
+
+.SH DESCRIPTION
+.B nm\-connection\-editor
+is a GTK\(hybased application to add, remove, and modify network connections
+stored by NetworkManager.  NetworkManager must be running for any network
+connections to be added, removed, or modified.
+
+.SH OPTIONS
+.TP
+.B \-t, \-\-type=<type>
+Type of connection to create or show.  Values are NetworkManager
+setting names, eg "802\-3\-ethernet", "802\-11\-wireless", "bridge", "bond",
+"wimax", "infiniband", "vlan", etc.  Does nothing if \-\-create or \-\-show
+is not given.
+.TP
+.B \-s, \-\-show
+Expand or collapse the network list to highlight the network connection type
+given by \-\-type.
+.TP
+.B \-c, \-\-create
+Create a new connection of the type given by \-\-type and allow the user to
+modify connection details.
+.TP
+.B \-e, \-\-edit=<uuid>
+Show the network connection edit window for the connection of the given UUID.
+
+.SH SEE ALSO
+.BR nmcli(1),
+.BR NetworkManager(8),
+.BR nm\-applet(1).
-- 
1.7.1

